packages/flutter/lib/src/rendering/object.dart

```diff
@@ -126,7 +126,8 @@ class PaintingContext {
   void paintChild(RenderObject child, Offset offset) {
     assert(() {
       if (debugProfilePaintsEnabled)
-        Timeline.startSync('${child.runtimeType}');
+        Timeline.startSync('${child.runtimeType}',
+          arguments: const <String, String>{ 'mode': 'basic' });
```

Comment:
nit: if blocks with multiple lines need braces.

Also, please use Flutter-style line wrapping (newline after the `(`, newline before the `)`) if you wrap to multiple lines.

In this case what you might want to do is declare this constant in foundation/debug.dart, and just use that here. That'll avoid a lot of duplicate code and also will let you not wrap things to two or more lines.

Reply:
done